nsdb-http/src/main/scala/io/radicalbit/nsdb/web/routes/QueryApi.scala

```diff
@@ -164,7 +169,13 @@ trait QueryApi {
                 case Some(statement) =>
                   onComplete(readCoordinator ? ExecuteStatement(statement)) {
                     case Success(SelectStatementExecuted(_, values)) =>
-                      complete(HttpEntity(ContentTypes.`application/json`, write(QueryResponse(values))))
+                      qb.parsed match {
```

Comment:
I think is enough something like this:
`complete(HttpEntity(ContentTypes.`application/json`, write(QueryResponse(values, qb.parsed.map(_ => statement)))))`

Reply:
that's cool!